flexmeasures/data/models/reporting/tests/test_aggregator.py

```diff
@@ -0,0 +1,37 @@
+import pytest
+
+from flexmeasures.data.models.reporting.aggregatator import AggregatorReporter
+
+from datetime import datetime
+from pytz import utc
+
+
+@pytest.mark.parametrize(
+    "aggregation_method, expected_value", [("SUM", 2), ("MEAN", 1)]
+)
+def test_aggregator(setup_dummy_data, aggregation_method, expected_value):
+    """ """
+    s1, s2, reporter_sensor = setup_dummy_data
+
+    reporter_config_raw = dict(
+        beliefs_search_configs=[
+            dict(sensor=s1.id, source=1),
+            dict(sensor=s2.id, source=2),
+        ],  # TODO: make source compulsory
```

Comment:
Your PR description suggests this todo has been done.

Reply:
Oh, left over comment. Thanks for pointing it out.